platforms/documentation/docs/src/docs/release/notes-template.md

```diff
@@ -16,7 +17,9 @@ Be sure to check out the [public roadmap](https://blog.gradle.org/roadmap-announ
 
 Switch your build to use Gradle @version@ by updating your wrapper:
 
-`./gradlew wrapper --gradle-version=@version@`
+`SHA=$(curl https://services.gradle.org/distributions/gradle-@version@-bin.zip.sha256 -J -L)`
```

Comment:
❌ Unfortunately, that probably does not work as that file will not yet be deployed.

I think what we need to do here is make sure docs are built after the `-bin` distribution, add the option to select the bin distribution and have its checksum an input to docs building.

And we should make sure that when running this for a local docs test, a fake value is provided so building the `-bin` distribution is not always required.

Reply:
Removed for now.